showcase/src/app/shared/example-viewer/example-viewer.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { Observable } from "rxjs";
+
+@Injectable()
+export class ExampleViewerService {
```

Comment:
Maybe we can rename this service to something more generic that really describes what it does: fetching files.

I know this service is only used by the example viewer component for now but maybe we can use it later for other components/features.

Reply:
True, maybe we can call it FileService, so we could extend it easier if we want to add other methods linked to file manipulation